command/fmt_test.go

```diff
@@ -50,3 +54,82 @@ func TestFmt_unfomattedTemlateDirectory(t *testing.T) {
 		fatalCommand(t, c.Meta)
 	}
 }
+
+func TestFmt_Recursive(t *testing.T) {
```

Comment:
I think that it would be a good idea to also check here that the files are not recursively formatted in case recursive is false.